app/controllers/organisations_controller.rb

```diff
@@ -57,6 +57,15 @@ def show
 
 private
 
+  def policies
+    rummager_results = Whitehall.unified_search_client.unified_search(
+      filter_organisations: [@organisation.slug],
+      filter_format: "policy"
```

Comment:
I initially left this out as there were some tests that acted as though the appearance of the "more" link was dependent on the full count returned.

It's actually not, so I can put this back in.  The only annoyance is the test helpers use fixtures, so I'll have to regenerate and change those.